src/autoload.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+spl_autoload_register(function ($className) {
+
+    // Abort here if we do not try to load BTCPayServer namespace.
+    if (strpos($className, 'BTCPayServer') === false) {
```

Comment:
This is not a good idea. We should be strict about the class name starting with this substring so it should be `=== 0` instead of ` === false `. May need to add a `\` in front so `\BTCPayServer` but I'm not 100% sure.

Reply:
Good point, I ended up with `if (strpos($className, 'BTCPayServer') !== 0) {`